src/main/java/org/generationcp/middleware/dao/oms/CVTermRelationshipDao.java

```diff
@@ -270,6 +273,22 @@ public Integer retrieveAnalysisDerivedVariableID(final Integer originalVariableI
 		}
 	}
 
+	public List<String> getScaleCategoriesUsedInAttributes(final int scaleId) {
+		final SQLQuery query = this.getSession().createSQLQuery(
+			"SELECT v.name category "
```

Comment:
Wecan do "Select **DISTINCT** v.name category" so we don't have to a` group by v.name`

Reply:
done.